libarchive/archive_read_disk_posix.c

```diff
@@ -1244,6 +1250,38 @@ archive_read_disk_set_matching(struct archive *_a, struct archive *_ma,
 	return (ARCHIVE_OK);
 }
 
+int
+archive_read_disk_set_forced_mtime(struct archive *_a,
+	__LA_TIME_T mtime, char clamp)
+{
+	struct archive_read_disk *a = (struct archive_read_disk *)_a;
+	a->has_forced_mtime = 1;
+	a->clamp_forced_mtime = clamp;
+	a->forced_mtime = mtime;
+	return (ARCHIVE_OK);
+}
+
+int
+archive_read_disk_set_forced_mtime_str(struct archive *_a,
```

Comment:
Keeping this in `..._posix` will likely prevent bsdtar from linking properly on Windows. Is this feature intended to be specific to non-Windows platforms? It doesn't look platform-specific. 🙂